src/simba/review_daily_collections/includes/collections_table.html

```
@@ -0,0 +1,415 @@
+
+                <!-- Dropdown -->
+                {% include 'src/simba/partials/dropdowns/table_sort.html' %}
+                <!-- End Dropdown -->
+              </div>
+              <!-- End Sort Dropdown -->
+            </th>
+
+            <th scope="col">
+              <!-- Sort Dropdown -->
+              <div class="hs-dropdown relative inline-flex w-full cursor-pointer">
+                <button id="hs-pro-eptsks" type="button"
```

HTMLHint (1.5.0)

[error] The id value [ hs-pro-eptsks ] must be unique.

(id-unique)